style.css

```diff
@@ -115,6 +139,236 @@ h2 {
   gap: 1.25rem;
 }
 
-.social-media a :hover {
-  fill: #3909c8;
+.section-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-gap: 88px;
+  padding: 114px 24px;
+}
+
+.project-card {
+  max-width: 327px;
+  border-radius: 16px;
+  padding: 16px;
+  color: #344563;
+  margin: auto;
+  gap: 12px;
+  background-color: #fff;
+}
+
+.snapshot-portfolio-img {
+  width: 100%;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.left-block {
+  margin-top: 1rem;
+}
+
+.project-title {
+  font-size: 36px;
+  font-weight: 700;
+  margin-bottom: 1rem;
+  font-family: poppins, sans-serif;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.counter {
+  padding: 0;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+  color: #344563;
+}
+
+.canopy-text {
+  display: flex;
+  padding: 0;
+  justify-content: left;
+  gap: 1.25rem;
+  letter-spacing: 0.05em;
+  list-style-type: none;
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.primary-text {
+  font-family: poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.01em;
+  text-align: left;
+  color: #344563;
+}
+
+.programming-languages {
+  color: solid #000;
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  gap: 12px;
+  padding: 0;
+}
+
+.languages .programming-languages .btn:hover {
+  color: #a20c31;
+}
+
+.languages a {
+  border-radius: 8px;
+  padding: 4px 12px;
+  background-color: #ebebff;
+  font-family: poppins, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #6070ff;
+}
+
+.about-myself-section {
+  color: #344563;
+  max-width: 375px;
+  border-radius: 16px;
+  padding: 16px;
+  margin: auto;
+  gap: 12px;
+  background-color: #fff;
+}
+
+.text-see-project {
+  margin: 0.75rem 0;
+}
+
+.see-project-button {
+  padding: 12px;
+  border: solid 1px #396df2;
+  border-radius: 8px;
+  color: #396df2;
+}
+
+.text-see-project .see-project-button:hover {
+  box-shadow: 0 8px 16px pink;
+  color: #a20c31;
+}
```

Comment:
- [ ] As requested by the previous reviewer you did an amazing job adding a hover effect to your **See project button** 👏🏽, yet the outcome is a bit far from what you have in the design. Please take a look 👇🏽

### Your current design
![image](https://user-images.githubusercontent.com/43172164/206192962-8447c30d-8817-4f48-b29f-f53583fd418f.png)

### What you are aiming to achieve
![image](https://user-images.githubusercontent.com/43172164/206193189-cb5a7619-5f91-4fc2-b359-1c1f4aac6512.png)

**_Tip:_** Instead of changing the `box-shadow` consider changing the `background-color`, and set your text color to **white**.